pyomo/common/unittest.py

```diff
@@ -0,0 +1,152 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+import math
+import six
+
+# Import base classes privately (so that we hae handles on them)
+import pyutilib.th.pyunit as _pyunit
+from pyutilib.th.pyunit import unittest as _unittest
+
+# Now, import the base unittest environment.  We will override things
+# specifically later
+from unittest import *
+from pyutilib.th import *
```

Comment:
I will remind you - I did just get rid of the rest of the `import *` statement.

Reply:
I debated this for a long time.  The challenge is that this module should be "unittest with extra bells and whistles".  We can explicitly list the objects from `unittest.__all__`, but I worry about "drift" in the underlying unittest library, where different versions have different exported symbols.  Thoughts?

(The `from pyutilib.th import *` will disappear as we complete the separation of Pyomo/PyUtilib and the required parts of pyutilib.th are moved here)

Reply:
I'm fine with leaving this as-is. No need to do extra work re: pyutilib nonsense for now.